app/helpers/buildout_design_system/application_helper.rb

```diff
@@ -1,4 +1,18 @@
 module BuildoutDesignSystem
   module ApplicationHelper
+    def component(component_name, *args, **kwargs, &block)
+      render(infer_class(component_name).new(*args, **kwargs), &block)
+    end
+
+    def infer_class(class_name)
+      @@class_map ||= {}
+      return @@class_map[class_name] if @@class_map[class_name].present?
+
+      klass = "::BuildoutDesignSystem::#{class_name}".constantize
```

Comment:
If I recall correctly, `constantize` is a pretty slow method, so this method will memoize the results in a class variable